dcp_prototype/frontend/src/components/exploreData.js

```diff
@@ -1,19 +1,20 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import { Flex, Box, Text } from "theme-ui"
 import StyledButton from "./styledButton"
 import LoginSignup from "./login-signup"
 
 import { api_prefix } from "../globals"
 
 const ExploreData = ({ project, files, isAuthenticated }) => {
-  const cxg_url = "http://cellxgene-dev.us-west-2.elasticbeanstalk.com/"
-  const disabled = !((files && files.length && isAuthenticated) === true)
+  const cxgUrl= "http://cellxgene-dev.us-west-2.elasticbeanstalk.com/"
```

Comment:
In a future PR we should migrate this value to the [env.production](https://www.gatsbyjs.org/docs/environment-variables/) store in s3 using https://github.com/chanzuckerberg/dcp-prototype/blob/a97a3414490006f0cb60e1506052ac05db298611/dcp_prototype/frontend/Makefile#L28

Reply:
Y'all may want to parameterize these sooner than later. We should remove as many blockers to a production deployment as possible.

Reply:
https://app.zenhub.com/workspaces/dcp2-5e2a191dad828d52cc78b028/issues/chanzuckerberg/dcp-prototype/229